build.sh

```diff
@@ -206,35 +206,40 @@ if [ "$clangTidy" -eq 1 ] && command -v clang-tidy ; then
 
     grep '"file"' $compile_json | sed -e 's/"//g' -e 's/^\s*file\s*:\s*//' > $cpp_list
 
-    while read -r cpp ; do
-        # Example:
-        # clang-tidy-3.8 -p build_dev/compile_commands.json plugins/GSdx/GSTextureCache.cpp -checks='modernize-loop-convert' -fix
-        # List of modernize check
-        # modernize-loop-convert
-        # modernize-make-unique
-        # modernize-pass-by-value
-        # modernize-redundant-void-arg
-        # modernize-replace-auto-ptr
-        # modernize-shrink-to-fit
-        # modernize-use-auto
-        # modernize-use-default
-        # modernize-use-nullptr
-        # modernize-use-override
-
-        # Check all, likely severals millions of log...
-        #clang-tidy -p $compile_json $cpp -checks='*' -header-filter='.*'
-
-        # Don't check header, don't check google/llvm coding conventions
-        clang-tidy -p $compile_json "$cpp" -checks='*,-llvm-*,-google-*'  >> $summary
-    done < $cpp_list
+    if command -v parallel >/dev/null ; then
+        # Run clang-tidy in parallel with as many jobs as there are CPUs.
+        parallel clang-tidy -p $compile_json "{}" -checks='*,-llvm-*,-google-*' < $cpp_list > $summary
```

Comment:
Ah, but the shebang at the top of the file is `#!/bin/sh` (you changed it recently, which is why I finished the POSIXification in an earlier commit). Which means it will not load `.bashrc`. Moreover, when starting `/bin/sh` like this, that's a non-interactive shell, I do not believe that any `rc` file is loaded: https://www.gnu.org/software/bash/manual/html_node/Bash-Startup-Files.html

If it were true that user preferences in `.bashrc` or any other `*rc` file could have such an effect on scripts that aren't incredibly defensively written, very few things would work as expected.